Lib/fontParts/base/glyph.py

```diff
@@ -1763,6 +1763,10 @@ def _interpolate(self, factor, minGlyph, maxGlyph,
         """
         Subclasses may override this method.
         """
+        from fontMath.mathFunctions import setRoundIntegerFunction
+
+        setRoundIntegerFunction(normalizers.normalizeVisualRounding)
```

Comment:
Why this, rather than just using the normalizer directly?

Reply:
I was trying to be as minimally invasive as possible. The `.round()` function on a fontMath object rounds in multiple places, for instance on `MathGlyph`:

https://github.com/robotools/fontMath/blob/adc7c38b789e0e5d3cb9100abb2c38c9049a0cca/Lib/fontMath/mathGlyph.py#L253-L279

I thought it was easier to do this than to recreate that functionality.

Reply:
got it